models/metrics/metric_erc1155_mint_burn_count.sql

```diff
@@ -0,0 +1,68 @@
+{{ 
+    config(
+        materialized='incremental',
+        tags=['metrics'],
+        incremental_strategy = 'delete+insert',
+        unique_key=['day_date'],
```

Comment:
```suggestion
        unique_key=['day_date','evm_contract_address'],
```
test this, not sure if it will work.

Reply:
this also works